### DISCUSSION TITLE
Written Number detection using C#. How Can I solve this? plss help me

### DISCUSSION THREAD

**Discussion by randoze** [community/community#21385] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e0f6d6343605ada816e499e470f12f1c75594d8f.jpeg" data-download-href="/uploads/short-url/w67NvZmgUNTz2KZsdH03QzJk5K7.jpg?dl=1" title="raven.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46586599/181063962-b22eae08-fa4c-411a-896b-37ec8fa548d3.jpg" alt="raven.jpg" data-base62-sha1="w67NvZmgUNTz2KZsdH03QzJk5K7" width="670" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0f6d6343605ada816e499e470f12f1c75594d8f_2_670x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/e/e0f6d6343605ada816e499e470f12f1c75594d8f_2_1005x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e0f6d6343605ada816e499e470f12f1c75594d8f.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0f6d6343605ada816e499e470f12f1c75594d8f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">raven.jpg</span><span class="informations">1072×800 192 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/randoze">@randoze</a>,</p>
<p>This appears to be C+ homework. We want to help you get the information you need, but we also are concerned that just giving you the answer will rob you of the necessary skills you’ll need to develop. If you can describe the things you’ve tried and how they haven’t worked for you, perhaps someone can give you a nudge in the right direction.</p>
<p>Cheers!</p>

**Comment by randoze** [comment, marked as answer]
<p>what tools do i use to achieve this ?</p>

**Comment by fire-eggs** [comment]
<p>Suggest you investigate “Handwriting Recognition”. You don’t indicate your target platform; open-source or commercial software, so it is difficult to suggest libraries. For example, Microsoft has Microsoft.Ink and some sort of Azure service.</p>
